tools/verify_turboquant.py

```
@@ -0,0 +1,425 @@
+
+    return kv, rot_matrix, indices, norms
+
+
+def verify_decompress(indices, norms, centroids, rot_matrix, kv_orig):
+    """Verify decompress operator: centroid lookup + scale + inverse rotate."""
+    print()
+    print("=" * 70)
+    print("3. Decompress Operator Verification")
+    print("=" * 70)
+
+    N, D = indices.shape
```

Ruff (0.15.15)

[warning] Unpacked variable `N` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)